perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -308,27 +310,100 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			ac.stateChangingCheck();
 
 			ac.getGroupsManager().addMember(ac.getSession(),
-					ac.getGroupById(parms.readInt("group")),
-					ac.getMemberById(parms.readInt("member")));
+				ac.getGroupById(parms.readInt("group")),
+				ac.getMemberById(parms.readInt("member")));
 			return null;
 		}
 	},
 
+	/*#
+	 * Removes members from a group.
+	 *
+	 * @throw MemberNotExistsException When member doesn't exist
+	 * @throw NotGroupMemberException  When member is not in the group
+	 * @throw GroupNotExistsException When the group doesn't exist
+	 * @throw WrongAttributeAssignmentException When assigning atribute to wrong entity
+	 * @throw AttributeNotExistsException When attribute doesn't exist
+	 * @throw ExternallyManagedException When the group is externally managed
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param members List<Integer> Members - can be empty <code>id</code>
+	 */
+	removeMembers {
+		@Override
+		public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
+			ac.stateChangingCheck();
+			List<Integer> memberInts = parms.readList("members", Integer.class);
+			if (memberInts == null) {
+				throw new RpcException(RpcException.Type.MISSING_VALUE, "parameter 'members' not sent");
```

Comment:
As @stavamichal mentioned, result can be null, when caller sends param, but with the null value, instead of empty list. In such case ``"parameter 'members' not sent"`` is not exactly right. You should state, that we expect non-empty list. In case of list is empty, no member is removed from the group. This can be stated in the javadoc too.